qa/suites/upgrade/octopus-x/parallel/1-ceph-install/octopus.yaml

```diff
@@ -43,7 +43,7 @@ tasks:
       - ceph orch ls
       - ceph orch host ls
       - ceph orch device ls
-      - ceph orch apply mds --placement=host.a
+      - ceph orch apply mds cephfs --placement=host.a
```

Comment:
@yuriw You will need to remove roleless for it to work.

Reply:
@varshar16 I am not sure if we need to keep jumping over the hoops and workarounds or wait till mds issues are fixed?
joshd ^ ?

Reply:
@varshar16 well we tried an failed again:
https://pulpito.ceph.com/teuthology-2020-06-30_21:03:21-upgrade:octopus-x:parallel-master-distro-basic-smithi/

Reply:
Please remove `mds.a` from roles. As we are creating one with `orch apply`.

Reply:
Example: https://github.com/ceph/ceph/blob/master/qa/suites/rados/cephadm/workunits/task/test_orch_cli.yaml